### PR TITLE
Add SSL support

### PR DESCRIPTION
**Pull request by ktibi**
Draft mode : 

@jnonino This PR works on my AWS account, but I take your comments.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -47,13 +47,27 @@ module "ecs_fargate" {
   container_cpu                = 4096
   container_memory             = 8192
   container_memory_reservation = 4096
+  enable_autoscaling           = false
```
Comment (jnonino) [inline main.tf:50]: Can this be set using a variable?

File: main.tf [hunk 2]
```diff
@@ -47,13 +47,27 @@ module "ecs_fargate" {
...
     default = {
-      listener_port     = 80
+      listener_port     = 443
       target_group_port = 9000
+      target_group_protocol = "HTTP"
     }
   }
-  lb_https_ports = {}
+
+  lb_enable_cross_zone_load_balancing = true
```
Comment (jnonino) [inline main.tf:67]: Can this be set using a variable?

File: main.tf [hunk 3]
```diff
@@ -47,13 +47,27 @@ module "ecs_fargate" {
...
+  enable_autoscaling           = false
+
   lb_http_ports = {
+  #  default = {
+  #    listener_port     = 80
+  #    target_group_port = 9000
+  #  }
+  }
+
+  lb_https_ports = {
```
Comment (jnonino) [inline main.tf:59]: Is it possible to have this configuration with SSL as default but allow module users to disable and use http?

**Comment by jnonino** [comment]
Hi @ktibi, I posted some comments, it looks good to me!

**Comment by ktibi** [comment]
> Hi @ktibi, I posted some comments, it looks good to me!

I made changes.

**Comment by jnonino** [comment]
> 

Looks good to me, if it tested I can approve it as soon you are ready to set it as `Ready for review`

**Review by jnonino — APPROVED** [review]

**Comment by jnonino** [comment]
Hi @ktibi , I found this issue with the PR
image
Can you create a new PR with a fix please?

**Pull request merged**